.github/workflows/github-ci.yml

```
@@ -72,39 +60,62 @@ jobs:
+    if: always() && github.event_name == 'pull_request'
     steps:
-      - name: Define message and notify Discord
+      - name: Notify Discord of PR status
         env:
           DISCORD_BOT_TOKEN: ${{ secrets.DISCORD_BOT_TOKEN }}
           DISCORD_CHANNEL_ID: ${{ secrets.DISCORD_CHANNEL_ID }}
-          AUTHOR: ${{ github.actor }}
-          BRANCH: ${{ github.head_ref || github.ref_name }}
-          TARGET_BRANCH: ${{ github.base_ref || github.ref_name }}
           STATUS: ${{ needs.tests.result }}
         run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)